android-sdk/build.gradle

```diff
@@ -37,9 +38,16 @@ android {
 dependencies {
   compile fileTree(dir: 'libs', include: ['*.jar'])
 
+  compile 'com.madgag.spongycastle:core:1.54.0.0'
```

Comment:
What is spongy castle and how is it used within android-idk library?  I usually like to comment the gradle file for all 3rd-party libs.